torch/csrc/jit/tensorexpr/ir_printer.cpp

```diff
@@ -191,7 +189,7 @@ static void formatImm(std::ostream& os, T v) {
   if (std::isnan(v)) {
     os << "NAN";
   } else if (std::isinf(v)) {
-    os << (v > 0 ? "POS_INFINITY" : "NEG_INFINITY");
+    os << (v > 0 ? "INFINITY" : "-INFINITY");
```

Comment:
Looks like this change causes miscompilation on cuda:
```
Feb 23 10:57:36 ======================================================================
Feb 23 10:57:36 ERROR [2.194s]: test_clamp (__main__.TestTEFuser)
Feb 23 10:57:36 ----------------------------------------------------------------------
Feb 23 10:57:36 Traceback (most recent call last):
Feb 23 10:57:36   File "test_jit_fuser_te.py", line 434, in test_clamp
Feb 23 10:57:36     s = self.checkScript(f, (inp1, inp2), profiling=ProfilingMode.PROFILING)
Feb 23 10:57:36   File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/jit_utils.py", line 457, in checkScript
Feb 23 10:57:36     frames_up=2)
Feb 23 10:57:36   File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/jit_utils.py", line 482, in checkScript
Feb 23 10:57:36     opt_script_outputs = scripted_fn(*recording_inputs)
Feb 23 10:57:36   File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py", line 129, in prof_func_call
Feb 23 10:57:36     return prof_callable(func_call, *args, **kwargs)
Feb 23 10:57:36   File "/opt/conda/lib/python3.6/site-packages/torch/testing/_internal/common_utils.py", line 126, in prof_callable
Feb 23 10:57:36     return callable(*args, **kwargs)
Feb 23 10:57:36 RuntimeError: default_program(24): error: identifier "INFINITY" is undefined
Feb 23 10:57:36 
Feb 23 10:57:36 1 error detected in the compilation of "default_program".
```

Reply:
I'll change it back to POS_INFINITY and NEG_INFINITY, didn't realize it's used for CUDA. In C++, we only have INFINITY and -INFINITY though.